### ISSUE TITLE
Easily configure keyword highlighting strings

### ISSUE THREAD

**Issue by dcht00** [geany/geany#3476]
I'd like Geany to highlight keywords like "TODO" and "XXX". This is not the default (as in gedit, for example), and there's no configuration list for it, AFAIK.

**Comment by elextr** [comment]
Keywords are listed in filetype configuration files, `Tools->Configuration files->Filetype Configurations->.....`.  If you want you can add them.

**Comment by dcht00** [comment]
That's true! But is it possible to highlight keywords *within* comment blocks? Sorry I wasn't exact enough.

**Comment by elextr** [comment]
Ahh, that depends on the specific lexer for the language, but its unlikely that any consider the contents of comments in any way.

**Comment by elextr** [comment]
See also the Geany-plugins add-ons plugin "Tasks" capability.
